Scoop/Controller/ProfileViewController.swift

```diff
@@ -356,6 +370,25 @@ class ProfileViewController: UIViewController, ProfileViewDelegate {
         }
     }
     
+    func updateDriverProfile() {
+        getUserPreferences()
+        guard let driver = user,
+              let imageURL = driver.profilePicUrl,
+              let pronouns = driver.pronouns,
+              let grade = driver.grade,
+              let hometown = self.hometown else { return }
+    
+        profileImageView.sd_setImage(with: URL(string: imageURL), placeholderImage: UIImage(named: "emptyimage"))
+        nameLabel.text = "\(driver.firstName) \(driver.lastName)"
+        subLabel.text = "\(pronouns) • \(grade) • \(hometown)"
+        phoneLabel.text = user?.phoneNumber
+        talkativeSlider.value = self.talkative ?? 0
+        musicSlider.value = self.music ?? 0
```

Comment:
Wouldn't it be better to make the default value in the middle? Wait, I think you told me that the middle value is 0 but I don't remember lol.